mappings/net/minecraft/resource/ResourceReloadHandler.mapping

```diff
@@ -1,9 +1,28 @@
 CLASS xh net/minecraft/resource/ResourceReloadHandler
+	CLASS xh$a Creator
```

Comment:
This should be `Factory`, to be consistent with all the other `Factory` classes we have.